docs/framework/concepts/yaml_format.md

```diff
@@ -0,0 +1,283 @@
+---
+short-description: Understanding the YAML format used to create apps
+...
+# The YAML format
+
+The Modular framework provides tools for creating knowledge apps by combining existing modules. App creators can combine these modules and describe the visual structure of their apps by writing YAML files that follow a specific format. This document is meant to explain the format of this YAML file. Here's the fist example:
+
+```
+---
+root:
+  type: Controller.Mesh
+  slots:
+    window:
+      type: Window.Simple
+      slots:
+        content:
+          type: ContentGroup.StaticText
+          properties:
+            label: "Hello World"
+```
+
+This is the simplest working app that can be described using the YAML format. The app description follows a tree-like structure, always starting from a `root` node. Each node represents a module of the framework. The most basic components of a node are:
+
+* the `type` of the module to be used in that node.
+* the `properties` of the module.
+* the `slots` of that module.
+
+A real example:
+```
+---
+root:
+  type: Controller.Mesh
+  slots:
+    window:
+      type: Window.Simple
+      slots:
+        content:
+          type: Pager.Simple
+          slots:
+            home-page:
+              type: ContentGroup.ContentGroup
+              properties:
+                expand: true
+              slots:
+                arrangement:
+                  type: Arrangement.List
+                  properties:
+                    expand: true
+                  slots:
+                    card:
+                      type: Card.List
+                      properties:
+                        expand: true
+                selection:
+                  type: Selection.All
+                  slots:
+                    filter:
+                      type: Filter.Articles
+                    order:
+                      type: Order.Sequence
+```
+
+This example expands the previous one. Now the app is capable of handling pages and displays a list of articles sorted in a sequence, on its home page. Extracting one of the nodes to see its components:
+
+```
+                arrangement:
+                  type: Arrangement.List
+                  properties:
+                    expand: true
+                  slots:
+                    card:
+```
+
+This is a node named "arrangement", for a module type _Arrangement.List_, which sets a property named "expand", and only has 1 slot named "card". The `slots` keyword is used to add sub-nodes, and to construct the tree. The list of available `properties` and `slots` depends on each type of module. As shown in the full example above, we can create a wide variety of apps by combining and constructing modules trees.
```

Comment:
"arrangement", "expand", "card" -> use backticks instead of double quotes
Arrangement.List -> link to the page in the API reference
modules trees -> module trees